### DISCUSSION TITLE
Switch table theme to dark when changing to dark mode in quarto

### DISCUSSION THREAD

**Discussion by DrNickRedfern** [rstudio/gt#1526] [Q&A]
Is it possible to change the theme applied to a table prouduced using gt when toggling between light and dark modes when producing html output in quarto? This is possible when producing tables with kable - toggling between light and dark mode automatically changes the appearance of a table to be conistent with the theme. I haven't been able to find out if this is possible in gt or, if it is possible, how to do it.

**Comment by olivroy** [comment]
There is an open issue about this https://github.com/rstudio/gt/issues/1065
